Newsweek

paulgureghian opened the pull request:
Pull request for merging 'newsweek' branch into 'master'  branch.

ghost commented:
## DeepCode's analysis on [#7b2172](https://www.deepcode.ai/app/gh/paulgureghian/Newsweek.com/e4c5973f7a0cafdac3cdcafa85b61c6629fd0090/paulgureghian/Newsweek.com/7b21725762069cdd1238edd86e3038dbf44d6d24/_/%2F/code/?utm_source=gh_review) found:

- :x: **0** critical issues.  :warning: **0** warnings  and **0** minor issues.  :heavy_check_mark: **0** issues were fixed.

:speech_balloon: This comment has been generated by the [**DeepCode bot**](https://github.com/apps/deepcode-ci-bot/), installed by the owner of the repository. The DeepCode bot **protects** your repository by detecting and commenting on security vulnerabilities or other critical issues.

---
:relaxed: If you want to provide feedback on our bot, here is how to [contact us](https://www.deepcode.ai/feedback?select=4&utm_source=gh_review).

Redvanisation reviewed (changes requested):
# Changes required  ♻️ 

### Hello @paulgureghian   🌟

**Please implement the following in your project ⬇️** 

1. There is nothing on the website except for some navigation links, you need to clone the original website with everything in it (images, text, colors ...etc), this is how your project is looking right now:
<img width="1627" alt="Screen Shot 2019-11-21 at 9 37 26 PM" src="https://user-images.githubusercontent.com/33462180/69393584-93fb1980-0ca7-11ea-8a42-25ed5e39b906.png">


2. Make the README file descriptive, check [this](https://github.com/othneildrew/Best-README-Template) link for reference.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

imhta reviewed (commented):
# Marking as invalid ⚠️ 
HeIlo @paulgureghian, am marking this code review invalid because this project is incomplete
- Complete the project before sumbiting for another code review.
- Should have a descriptive README.md
----
### Don't forgot to make another code review request
----
Happy coding
**Pon Muthu Selvam N**
_If you have queries feel free to contact me on slack Pon Muthu Selvam_

trox115 commented:
### Invalid Submission 
Hey @paulgureghian , it was very nice to meet you :handshake: . Unfortunately, your project it's not ready for review. Please always remember that you can not submit a PR to ask for an opinion only when you think that the project is finished. 
If you need help with something, just reach me through slack. 

You got this :muscle:

imhta reviewed (commented):
# Marking as invalid ⚠️ 
HeIlo @paulgureghian, am marking this code review invalid because this project is incomplete
- Linter should pass currently your linter is failing. fix that
- Should have a descriptive README.md
----
### Don't forgot to make another code review request
----
Happy coding
**Pon Muthu Selvam N**
_If you have queries feel free to contact me on slack Pon Muthu Selvam_

paulgureghian committed:
Merge branch 'newsweek' of https://github.com/paulgureghian/Newsweek.com into newsweek

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited genericons.css file

paulgureghian committed:
Edited genericons.css file

paulgureghian committed:
Edited .stickler.yml file

paulgureghian committed:
Edited genericons.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited README.md file

paulgureghian committed:
Edited README.md file

paulgureghian committed:
Edited README.md file

paulgureghian committed:
Added image files

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited stlyes.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited styles.css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Added images

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

mosesogwo commented on styles.css:
```diff
@@ -0,0 +1,732 @@
+
+body {
+  margin-top: 0;
+  padding-top: 0;
```
Can be replaced with bootstrap classes.

mosesogwo commented on styles.css:
```diff
@@ -0,0 +1,732 @@
+
+.main-navigator a {
+  color: black;
+  font-weight: bold;
```
Can be replaced with a bootstrap class

mosesogwo commented on styles.css:
```diff
@@ -0,0 +1,732 @@
+h5,
+h6 {
+  font-family: "Roboto Condensed", sans-serif;
+  font-weight: bold;
```
Can be replaced with a bootstrap class

mosesogwo commented on styles.css:
```diff
@@ -0,0 +1,732 @@
+  padding-left: 6px;
+  padding-right: 6px;
+  margin-right: 6px;
+  margin-bottom: 6px;
```
May be replaced with a bootstrap classes

mosesogwo commented on styles.css:
```diff
@@ -0,0 +1,732 @@
+  font: bold 19px/19px "Roboto Condensed", sans-serif;
+  margin-bottom: 15px;
+  text-transform: uppercase;
+  align-items: center;
```
You can use a bootstrap class

mosesogwo commented on styles.css:
```diff
@@ -0,0 +1,732 @@
+  text-transform: uppercase;
+  align-items: center;
+  position: relative;
+  display: flex;
```
You can use a bootstrap class here

mosesogwo reviewed (changes requested):
### :clap:  Nice work. However, we need to make a few adjustments before we :airplane: 
To make your clone a proper replica:
1. Let us have the white space above the header removed.
![image](https://user-images.githubusercontent.com/46611870/71613953-60b28100-2ba9-11ea-9928-047b6788a6cf.png)

2. The editions section in the footer is not properly aligned.
- Yours look like this:
> ![image](https://user-images.githubusercontent.com/46611870/71614007-bdae3700-2ba9-11ea-9ac0-455d25d67fe0.png)

- The original looks like this:
> ![image](https://user-images.githubusercontent.com/46611870/71614099-20073780-2baa-11ea-9db3-76e33ecfa735.png)

3. Let us also remove this extra shade that appears on the subscribe button:
> 
![image](https://user-images.githubusercontent.com/46611870/71614194-9e63d980-2baa-11ea-8a76-33ce4e65d056.png)

4. The header is not properly aligned on  smaller screens:
 - Yours look like this:
> 
![image](https://user-images.githubusercontent.com/46611870/71614443-f3541f80-2bab-11ea-8cb7-58f17e891a6d.png)

- The original looks like this:
> 
![image](https://user-images.githubusercontent.com/46611870/71614497-4ded7b80-2bac-11ea-8484-1c4c14b36453.png)

5. You are required to use bootstrap classes everywhere possible. See comments on your stylesheet. Replace all similar stylings with bootstrap classes.
> 
![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/using_bootstrap/bootstrap_classes_requirement.png)

**Kindly make the changes and submit another code review request** .

## Status: Changes requested.

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited html file

paulgureghian commented:
Moses, the first two items I was able to fix. 
My header seems to align on my 8" android tablet, what size screen are you using ? 
As for the gray shade on the subscribe button, can you hint me how to fix it ?

nikasulo reviewed (changes requested):
# Under Review by Tolulope Olusakin(TSE)
You have done an amazing job! Many improvements since your first review, you only have a few more changes to make and you're done. Please attend to the following :handshake: 

#### Descriptive PR
A descriptive PR is necessary to give anyone who is looking at your work an idea of what has been done. Please consider describing the work you've done leading up to the PR in as much detail as possible in subsequent PRs.

#### Navigation bar
Your current navigation bar doesn't resize properly when moving to smaller screens. Please take a look below at the screenshots I've attached to view some of the issues. Consider using your developer tools to check the responsiveness of your page before your next review, it provides a lot of screen sizes to play with!
![screenshot](https://thumbs2.imgbox.com/d0/10/9yojuVxS_t.png)

#### Footer
Your footer looks real nice, however, its elements are not aligned properly in comparison with that of the original site, please give it a look, compare thoroughly and make the appropriate changes
![screenshot](https://thumbs2.imgbox.com/8a/c9/Ewp8DMn3_t.png)

#### Readme File
Your readme file looks great, however, it could use some finishing touches, please consider this [example](https://github.com/Oluwadamilareolusakin/bookstore-react-redux/blob/master/README.md) as a guide for improving yours! 

#### Horizontal scroll
You have some sort of margin or padding setup that is negatively impacting the flow of the page and causing it to scroll horiizontally. Please see a screenshot below.
![sceenshot](https://thumbs2.imgbox.com/f3/86/kHIyWqiE_t.png)
![screenshot](https://thumbs2.imgbox.com/82/64/v97CE5aL_t.png)
![screenshot](https://thumbs2.imgbox.com/f1/02/k4GJaCs9_t.png)

#### Responsiveness
The general responsiveness of the page needs a little improvement, the elements currently don't resize properly in relation to different screen sizes. As stated above, consider using your developer tools to test it's responsiveness before your next review. If you have Google Chrome, simply press CTRL + Shift + I` to bring up these tools. See a screenshot below of what it may look like.
![screenshot](https://thumbs2.imgbox.com/d7/0d/55JGhoiF_t.png)

### Summary
Your work is gradually coming to perfection, please look into the few things I've listed above, make changes and request another review. Best of luck!

paulgureghian commented:
Tolulope Olusakin(TSE)  Did you review my code ? I don't see your review.

imhta reviewed (changes requested):
# Changes required 🛑 

### Hello  @paulgureghian , 👋 You have done a good job in this project 👏 


### **under 990px width Yours**

- [ ] fix nav section 

![image](https://user-images.githubusercontent.com/25448577/71658935-b0d94100-2d6b-11ea-8656-96aef56a017a.png)

- [ ]  in the magazine section, it's layout is not same as original even in larger screen size fix that too

![image](https://user-images.githubusercontent.com/25448577/71659020-feee4480-2d6b-11ea-9d0f-52f4c3080e1d.png)

- [ ] alignment of authors in the options section is not the same

![image](https://user-images.githubusercontent.com/25448577/71659143-7f14aa00-2d6c-11ea-8e3d-a099f06da401.png)

### **under 990px Original**
![image](https://user-images.githubusercontent.com/25448577/71658980-dc5c2b80-2d6b-11ea-9a03-a8a83efc4650.png)
![image](https://user-images.githubusercontent.com/25448577/71659074-2fce7980-2d6c-11ea-91a2-d86ae200cb1a.png)

![image](https://user-images.githubusercontent.com/25448577/71659114-66a48f80-2d6c-11ea-80b5-15d1e881eac9.png)


----
### Don't forget to request another code review
----
**Pon Muthu Selvam N**
__nponmuthuselvam@gmail.com__
_If you have any queries, feel free to ping me on slack_

nikasulo commented:
# Hello @paulgureghian, I was in the process of going over your work and writing the review. Please see it above.

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian commented:
1. I was able to remove the whitespace abover the header.
2. I was able to align the editions section
3. I can't figure out how to remove the gray from the subscribe button
4. I was able to align the author names
5. I still need to fix the navbar and 'in the magazine' section
6. Can you hint me on how to fix them ?

paulgureghian committed:
Edited tml file

MauricioRobayo commented on styles.css:
```diff
@@ -0,0 +1,748 @@
+  text-decoration: none;
+  padding-top: 30px;
+  padding-left: 15px;
+  color: #d9d9d9;
```
The gray is from the default `background-color` of the button, set by the browser. You can overwrite it here, for example, you could add:  `background-color: transparent;`

paulgureghian replied:
That worked, although I am not sure why. I thought this .genericons class only accessed the social media icons in the footer.

MauricioRobayo commented on index.html:
```diff
@@ -0,0 +1,2688 @@
+        </div>
+
+        <div class="row">
+          <div class="col-12 col-md-3">
+            <div class="cover">
+              <div class="picture-frame">
+                <div class="picture-frame-image">
+                  <a href="#">
+                    <picture>
+                      <img
+                        class="img-fluid"
+                        src="images/nov_15_issue.webp"
+                        alt="Magazine Cover"
+                      />
+                    </picture>
+                  </a>
+                </div>
+                <div class="picture-frame-overlay">
+                  <a href="#">NOVEMBER 15 ISSUE</a>
+                </div>
+              </div>
+              <a class="see-all-features" href="#">SEE ALL FEATURES ></a>
+            </div>
+          </div>
```
Please take a look a the source code of the original page to get a better idea on how to structure this.

Notice on the source code of the original page that this first article is in the same `row` than the remaining three. Take some time to explore this section using the devtools of your preferred browser, that might give you the best idea on how they are doing it so you can try to mimic the layout:

![image](https://user-images.githubusercontent.com/2121481/71727152-64dbda00-2e07-11ea-9642-d9d48fec47df.png)

paulgureghian replied:
I am having trouble with this one.

MauricioRobayo reviewed (changes requested):
Hi @paulgureghian!

Truly admire your work ethic, you have done some incredible progress here.

* [x] I was able to remove the whitespace above the header.
* [x] I was able to align the editions section
* [x] I was able to align the author names
* [ ] I still need to fix the navbar and 'in the magazine' section 👈
* [ ] I can't figure out how to remove the gray from the subscribe button 👈

Please take a look at the comments I left on your code to see if they can help you with the remaining items.

You are almost there, keep up that level of energy 👩‍💻.

**Status**: Request changes.

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian commented:
1. I was able to remove the gray box from the subscribe button in the header in the .genericons class
   for proper display on smaller screens, although I thought that .genericons class was just for the 
  social media icons in the footer. 

2. As for the re-positioning of the first element in the 'in-the-magazine' section, 
   I have tried different things, but still can't figure it out.

nikasulo reviewed (changes requested):
# Under Review by Tolulope Olusakin(TSE)
Hello @paulgureghian, you're making more progres.  Please take a look at the comments from this below. It's the one I made earlier and the comments are still very applicable.

You're doing a good job and I'm sure you're learning a lot as well! Keep it up :smile: 

> # Under Review by Tolulope Olusakin(TSE)
> You have done an amazing job! Many improvements since your first review, you only have a few more changes to make and you're done. Please attend to the following 
> 
> #### Descriptive PR
> A descriptive PR is necessary to give anyone who is looking at your work an idea of what has been done. Please consider describing the work you've done leading up to the PR in as much detail as possible in subsequent PRs.
> 
> #### Navigation bar
> Your current navigation bar doesn't resize properly when moving to smaller screens. Please take a look below at the screenshots I've attached to view some of the issues. Consider using your developer tools to check the responsiveness of your page before your next review, it provides a lot of screen sizes to play with!
> ![screenshot](https://camo.githubusercontent.com/81b8618e9e2e4df79e423a13e45b5bb7585efea8/68747470733a2f2f7468756d6273322e696d67626f782e636f6d2f64302f31302f39796f6a755678535f742e706e67)
> 
> #### Footer
> Your footer looks real nice, however, its elements are not aligned properly in comparison with that of the original site, please give it a look, compare thoroughly and make the appropriate changes
> ![screenshot](https://camo.githubusercontent.com/3c5a36f88a0bc70a5917b6985fb4b8d6ac1c73b8/68747470733a2f2f7468756d6273322e696d67626f782e636f6d2f38612f63392f45777038444d6e335f742e706e67)
> 
> #### Readme File
> Your readme file looks great, however, it could use some finishing touches, please consider this [example](https://github.com/Oluwadamilareolusakin/bookstore-react-redux/blob/master/README.md) as a guide for improving yours!
> 
> #### Horizontal scroll
> You have some sort of margin or padding setup that is negatively impacting the flow of the page and causing it to scroll horiizontally. Please see a screenshot below.
> ![sceenshot](https://camo.githubusercontent.com/67a1638aa41f82acb371b769c6855b9e823a4b3b/68747470733a2f2f7468756d6273322e696d67626f782e636f6d2f66332f38362f6b484979577169455f742e706e67)
> ![screenshot](https://camo.githubusercontent.com/23e2ba8463fa30ed4e36fdd55b251c8e9cfb1a30/68747470733a2f2f7468756d6273322e696d67626f782e636f6d2f38322f36342f763937434535614c5f742e706e67)
> ![screenshot](https://camo.githubusercontent.com/3e907c007c29998541b4295928aa4603e1f9295b/68747470733a2f2f7468756d6273322e696d67626f782e636f6d2f66312f30322f6b34474a614373395f742e706e67)
> 
> #### Responsiveness
> The general responsiveness of the page needs a little improvement, the elements currently don't resize properly in relation to different screen sizes. As stated above, consider using your developer tools to test it's responsiveness before your next review. If you have Google Chrome, simply press CTRL + Shift + I` to bring up these tools. See a screenshot below of what it may look like.
> ![screenshot](https://camo.githubusercontent.com/d695aae3226b2cb068dd58b48a4fbe02b6838e7a/68747470733a2f2f7468756d6273322e696d67626f782e636f6d2f64372f30642f35354a47686f69465f742e706e67)
> 
> ### Summary
> Your work is gradually coming to perfection, please look into the few things I've listed above, make changes and request another review. Best of luck!

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited html file

paulgureghian commented:
I was able to re-position the first element in the  'in-the-magazine'  section. 
I moved a closing </div> tag lower in the file and it worked.

nikasulo reviewed (changes requested):
# Nice improvements @paulgureghian!
We still need to look more closely at my comments from the last review.

### Summary
We need to fix the navigation, page responsiveness,  section and element sizings. You've come a long way and I am glad for you! Keep going :rocket:

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian commented:
I added four @media and screen queries, but I still can't get it to re-size. 

Can you hint me what I need to do ?

nikasulo reviewed (changes requested):
# Nice improvements @paulgureghian! :rocket: 
Please lets meet in Slack so we can discuss your project based on your above request! Looking out for your message :100: 

Oluwadamilare Olusakin(TSE) :rocket:

paulgureghian committed:
Added logo and title

paulgureghian committed:
Deleted

paulgureghian committed:
Delete desktop.ini

Deleted.

paulgureghian committed:
Added code comments

paulgureghian committed:
Added code comments

paulgureghian committed:
Rewrote the header & navbar

paulgureghian committed:
Styled the header

paulgureghian committed:
Edited stylesheet link

paulgureghian committed:
Added two styles sheets

paulgureghian committed:
Deleted two css files

paulgureghian committed:
Fixed linter issues

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Rewrote the header and nav

paulgureghian committed:
Styled the header and navbar

paulgureghian committed:
Styled the main and footer

paulgureghian committed:
Edited genericons file

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Added @import for Genericons

paulgureghian committed:
Added @import for Genericons

paulgureghian committed:
Edited the footer

paulgureghian committed:
Made minor changes

paulgureghian committed:
Styled the footer

paulgureghian committed:
Fixed linter issues

paulgureghian commented:
I fixed the body background color, made the header / navbar sticky, added background color to the main, fixed some images from being too big. and realigned the footer section.

I added some breakpoints, spruced up the html and css files with some code comments, validated the html, and fixed stickler errors. 

I think it's still not responding the way it should on the RDM emulator in FF dev tools. maybe you can hint me what needs to be done ?

Thanks.

uryelah commented:
Hello **Paul**, I'm Sarah, your reviewer for this PR 👩‍🚀  

First of all, the responsiveness of the page is actually quite good. 👏
I really liked how you also added hover interactions and the articles that have text over them are looking pretty good.

As this project requires great attention to detail and needs to be quite similar to the [original site](https://www.newsweek.com/), there are still issues to fix before moving on. You got this!💪 

---------    

### Required changes 🛠 
---------

- [ ] You have an unwanted horizontal scroll problem. Remember to set the width of the direct children of the `<body>`, like the `<main>`, to 100%, and make sure that the body has a width of 100vw. Hiding the `overflow-x` could also help.

 - [ ] Remember to change `<main>` margin-top on resize to medium-size screens since the nav height becomes smaller.

Your version with the extra light grey space:
![Screenshot_2020-03-22 Newsweek - News, Analysis(1)](https://user-images.githubusercontent.com/16193212/77262329-dba0aa80-6c73-11ea-9acb-5d849f80b69a.png)
Original one without empty space:
![Screenshot_2020-03-22 Newsweek(32)](https://user-images.githubusercontent.com/16193212/77262330-dba0aa80-6c73-11ea-8658-92f8a0a354b2.png)

----------


- [ ] Some of your images are 'glued' to each other. Remember to add some margin to them. [Check this for more on Bootstrap 4 spacing.](https://getbootstrap.com/docs/4.4/utilities/spacing/)
**Your 'In the Magazine' bottom row images**
![Screenshot_2020-03-22 Newsweek - News, Analysis(4)](https://user-images.githubusercontent.com/16193212/77262312-d6436000-6c73-11ea-9331-50ca57e3d457.png)
**Original version:**
![Screenshot_2020-03-22 Newsweek(34)](https://user-images.githubusercontent.com/16193212/77262314-d6dbf680-6c73-11ea-8375-26b5a3169dc3.png)
---------

- [ ] All the sections above 'In the magazine', in wide screens, are aligned at the bottom. You have some white space in some of them. Try to fix this with a `overflow: scroll` property for the longer sections or `overflow: hidden`.

![Screenshot_2020-03-22 Newsweek - News, Analysis(3)](https://user-images.githubusercontent.com/16193212/77262311-d5aac980-6c73-11ea-875b-f3d22e80e5f2.png)

**Original version (note the lack of white spaces):**
![Screenshot_2020-03-22 Newsweek(33)](https://user-images.githubusercontent.com/16193212/77262332-dc394100-6c73-11ea-980d-008ddd5b5e91.png)

------

- [ ] Your 'subscribe' section shade of grey is too dark in comparison to the original. Compare your and the one from the original site bellow. Tip: [use your browser eyedropper](https://developer.mozilla.org/en-US/docs/Tools/Eyedropper) to get the exact color you need.
![Screenshot_2020-03-22 Newsweek - News, Analysis(5)](https://user-images.githubusercontent.com/16193212/77262315-d7748d00-6c73-11ea-851e-53f8f45df5a1.png)
![Screenshot_2020-03-22 Newsweek(35)](https://user-images.githubusercontent.com/16193212/77262316-d7748d00-6c73-11ea-8f83-fca38fd4c765.png)


-----

- [ ] Your 'In the Magazine' images are too tall on medium screen resizes. Compare your version and the original bellow:
![Screenshot_2020-03-22 Newsweek - News, Analysis(6)](https://user-images.githubusercontent.com/16193212/77262318-d80d2380-6c73-11ea-9741-a5cded331dc6.png)
![Screenshot_2020-03-22 Newsweek(36)](https://user-images.githubusercontent.com/16193212/77262319-d80d2380-6c73-11ea-95e1-340d210b4c71.png)


---

- [ ] The images at the `.minor` divs are too small when resizing to small screens. Compare the image sizes in your version and the original site bellow:
![Screenshot_2020-03-22 Newsweek - News, Analysis(7)](https://user-images.githubusercontent.com/16193212/77262321-d93e5080-6c73-11ea-8e54-8cea9894a8a2.png)
![Screenshot_2020-03-22 Newsweek(37)](https://user-images.githubusercontent.com/16193212/77262322-d93e5080-6c73-11ea-9b03-c08bbff9347a.png)


---------

- [ ] In small screens most of the pictures in the site take 100% of the width of the page, minus the margins and paddings, in your version the width is lacking. This can be fixed easily with a `width: 100%` or similar. Please compare your travel image on small devices and the one from the original site:
![Screenshot_2020-03-22 Newsweek - News, Analysis(8)](https://user-images.githubusercontent.com/16193212/77262323-d9d6e700-6c73-11ea-8fc4-432d33a21bb5.png)
![Screenshot_2020-03-22 Newsweek(38)](https://user-images.githubusercontent.com/16193212/77262326-da6f7d80-6c73-11ea-8ecd-314cbce80799.png)


--------

- [ ] On medium and small devices the navbar should still show the weather information and icon. Compare your version and the original below:
![Screenshot_2020-03-22 Newsweek - News, Analysis](https://user-images.githubusercontent.com/16193212/77262327-db081400-6c73-11ea-9cd6-18ca60672192.png)
![Screenshot_2020-03-22 Newsweek(31)](https://user-images.githubusercontent.com/16193212/77262328-db081400-6c73-11ea-9949-885ed55441d7.png)

------

You are doing a good job with this project Paul, keep it up with the improvements and it will be an almost exact clone of the original page 😉 

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

paulgureghian committed:
Linked in media.css

paulgureghian committed:
Moved media queries to media.css

paulgureghian committed:
Moved media queries to media.css

paulgureghian committed:
Added media.css

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Made minor edits

paulgureghian committed:
Minor edit to 'head' tag

paulgureghian committed:
Added media queries

paulgureghian committed:
Added media queries

paulgureghian committed:
Added two images

paulgureghian committed:
Added various selectors to breakpoints

paulgureghian committed:
Styled advertisements class

paulgureghian committed:
Added weather widget

paulgureghian commented:
Required changes 🛠
1. You have an unwanted horizontal scroll problem. Remember to set the width of the direct children of the <body>, like the <main>, to 100%, and make sure that the body has a width of 100vw. Hiding the overflow-x could also help.

I think I fixed this one. 

2. Remember to change <main> margin-top on resize to medium-size screens since the nav height becomes smaller.

I tried to fix this and I couldn't seem to access the <main>  here:

/* MEDIUM DEVICES */
@media only screen and (min-width: 768px) {
  header .logo svg {
    width: 140px;
  }

  main {
    margin-top: 133px;
  }
  
3. Some of your images are 'glued' to each other. Remember to add some margin to them. Check this for more on Bootstrap 4 spacing.

I tried different BS classes like  ml, mr, mx, pl, pr, px  and 1,2,3,4,5 and auto, and different div placements and I couldn't figure it out. 

<div class="col-md-3 col-12">
                   <div class="mx-3"> 
                    <a href="#">
                      <picture>
                        <img
                          src="images/magazine_6.webp"
                          alt="In The Magazine"
                        />
                      </picture>
                    </a>
                    <div class="tags">
                      <span class="main-tag">DOWNTIME</span>
                      <a href="#"><span class="secondary-tag">CULTURE</span></a>
                    </div>
                    <h3>
                      <a href="#"
                        >Kelly McCreary on the Writers of 'Grey's Anatomy' and
                        Getting Out the Vote</a
                      >
                    </h3>
                   </div>
                  </div>

4. All the sections above 'In the magazine', in wide screens, are aligned at the bottom. You have some white space in some of them. Try to fix this with a overflow: scroll property for the longer sections or overflow: hidden.

I tried 

.advertisements {
  overflow-y: hidden;
} 

on this: 

<aside class="advertisements"> , it did not work. do I need a <div> instead ?

5. Your 'subscribe' section shade of grey is too dark in comparison to the original. Compare your and the one from the original site bellow. Tip: use your browser eyedropper to get the exact color you need.

Changed to  #f4f5f6.  this change was easy. 

6. Your 'In the Magazine' images are too tall on medium screen resizes. Compare your version and the original bellow:

Added to medium device breakpoint:
.cover picture img {
    top: 0;
    left: 50%;
    width: auto;
    height: 100%;
  }

  .in-the-magazine picture img {
    top: 0;
    left: 50%;
    width: auto;
    height: 100%;
  } 

But Doesn't seem to work. 

7. The images at the .minor divs are too small when resizing to small screens. Compare the image sizes in your version and the original site bellow:

Added this to small and xs device breakpoints:

.minor {
    width: 100%;
    height: auto;
    vertical-align: middle;
  }

The images seem bigger I think, but the rest of the section is still wrong I think.

8. In small screens most of the pictures in the site take 100% of the width of the page, minus the margins and paddings, in your version the width is lacking. This can be fixed easily with a width: 100% or similar. Please compare your travel image on small devices and the one from the original site:

Added this to xs and small media query, not sure if it worked, I don't think it did. 

.culture-and-travel-entry picture img {
    width: 100%;

9.  On medium and small devices the navbar should still show the weather information and icon. Compare your version and the original below:

I added a weather widget and it seems to display on small / medium devices, though I think the header / navbar is still not resizing properly.

Can you hint me on how to make further changes ?  
Thanks.

paulgureghian committed:
Added a weather widget

paulgureghian committed:
Made a minor edit

paulgureghian committed:
Styled the weather widget

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Fixed linter errors

mikenath223 commented:
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 🐱

mikenath223 commented:
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 🐱

mikenath223 reviewed (changes requested):
### Good job @paulgureghian 👌👏

You've done quite well laying out the page and adding subtle transition effects. **They look nice. 👌**

However, there are major changes that need to be made before proceeding. Kindly take note of them:👇

- [ ] **Required** - Fix linting errors in your stylesheet. You need this mark ❌ beside your commit to turn to this mark.✅ Kindly take note of this before requesting a new review. 

- [ ] Required - Remove comments in your code. If you want to refer to a previous version of your code use git history.

- [ ]  **REQUIRED** - I would ask that before you begin fixing the responsiveness issue on this project, you head over to the [odin lesson](https://www.theodinproject.com/courses/html5-and-css3/lessons/responsive-design) on responsiveness and complete the lessons together with the assignment. Then when you have satisfactory understood the lesson and been able to answer these questions below👇 then you begin working on the responsiveness issue on this project.![ice_screenshot_20200326-011309](https://user-images.githubusercontent.com/33323077/77598128-4ca8c200-6f01-11ea-8aed-35b84875a77f.png).


Hoping you have gone through the lesson once more, here is a breakdown of the issues on the page:

NOTE | SCREEN-SIZES
------ | ---------------
Notice how the original webpage looks on a tablet device👉 | TABLET DEVICE
If you observe the screenshot to the right carefully, you will notice that the original webpage doesn't have an x-axis scroll bar that scrolls the page to the right and also **all contents still fit into the page,** this is referred to as responsiveness.  | ![ice_screenshot_20200326-013552](https://user-images.githubusercontent.com/33323077/77598508-46ffac00-6f02-11ea-96b1-4c379d7dc5fe.png)
Yet this is how yours looks still on a tablet device 👉 | ![newsyours](https://user-images.githubusercontent.com/33323077/77599364-b4acd780-6f04-11ea-8559-0cb039facf58.png) It shouldn't look this way, a scrollbar along the x-axis exists, and all contents do not fit on the page unless I scroll. There should be no scrollbar and all contents should fit
---------------- | -------------------
Now notice again how the original webpage looks on a mobile device 👉 | MOBILE DEVICE
If you observe carefully the screenshot to the right, you will notice that all contents once again fit into the screen and there is once more no scrollbar along the x-axis. |![ice_screenshot_20200326-013458](https://user-images.githubusercontent.com/33323077/77599603-8aa7e500-6f05-11ea-908d-8a6593097707.png)
Yet this is how yours looks on a mobile device 👉 | ![ice_screenshot_20200326-015256](https://user-images.githubusercontent.com/33323077/77599688-d195da80-6f05-11ea-899f-bd41a7e29f78.png) You will notice that in yours, not only does the top navbar seem disoriented, also the contents of the page don't fit into the page as can be indicated by the scrollbar along the x-axis.

I do hope this was explicit enough in helping you understand the requirements of this project. 

Kindly fix this issues, and request another review.

**Once more I love your transition effects, they are subtle and cool 👌**

Feel free to reach out to me on slack if you need more clarification.

**Am rooting for you to get this done, I know you can 💪**

HAPPY CODING 👨‍💻

paulgureghian committed:
Added rules to body tag

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Fixed linter errors

paulgureghian committed:
Added body tag rules

paulgureghian commented:
Required changes:
If you observe the screenshot to the right carefully, you will notice that the original webpage doesn't have an x-axis scroll bar that scrolls the page to the right and also all contents still fit into the page, this is referred to as responsiveness.

If you observe carefully the screenshot to the right, you will notice that all contents once again fit into the screen and there is once more no scrollbar along the x-axis.

1. I fixed the horizontal scrolling issues on small and medium devices
   by adding some <body> rules in media.css
  
   It seems to have worked, but I think there are more issues to fix ?
   Please hint me as to what else needs to be done and how to do it. 

Thanks.

davidmrtz-dev commented:
## Hi @paulgureghian . 👨‍🚀  
The changes you did from the last review about the horizontal scrolling did not work.
Still have the same issues pointed by the previous TSE.
As the main point of this project is to use Bootstrap I'll recommend you don't use overflow property, instead use the bootstrap utilities to get the expected behavior and appearance.
Here's a couple of useful links where you can find information about the behavior of bootstrap on responsiveness and how to use their utilities. ⬇️ 
* Bootstrap [Grid System ](https://getbootstrap.com/docs/4.0/layout/grid/) 
* Bootstrap [Flex ](https://getbootstrap.com/docs/4.4/utilities/flex/) 
 👨‍💻   

****

### **Please kindly implement the changes required by the previous TSE and submit for re-review once you are ready!** 🙏 

****

### Status: Changes required. ♻️  👀 
****

Happy coding! 😄 💻  
If you have any questions or need help, you can message me on slack (@ David Eli Martinez Garcia).

**[TSE:  David Elí]** 👨‍💻 
Github: [@davitomix](https://github.com/davitomix)
Twitter: [@davitomix](https://twitter.com/davitomix007)
Linkedin: [David Elí Martínez García](https://www.linkedin.com/in/davideligarcia/)

paulgureghian committed:
Made a minor edit
